texar/torch/modules/pretrained/pretrained_bert.py

```diff
@@ -39,19 +39,34 @@ class PretrainedBERTMixin(PretrainedMixin, ABC):
     The BERT model was proposed in
     `BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding`_
     by `Devlin et al.` It is a bidirectional Transformer model pre-trained
-    on a large corpus. The available BERT models are as follows:
-    `bert-base-uncased`, `bert-large-uncased`, `bert-base-cased`,
-    `bert-large-cased`, `bert-base-multilingual-uncased`,
-    `bert-base-multilingual-cased`, and `bert-base-chinese`.
+    on a large corpus.
+
+    The available BERT models are as follows:
+
+      * `bert-base-uncased`: 12-layer, 768-hidden, 12-heads, 110M parameters.
```

Comment:
Let's change the names to using double backticks so they stand out from the descriptions, and also to indicate that the names are what should be provided as argument values.